enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -59,13 +59,19 @@ requests to off-cluster services, and those that use a
   + [Additional certificate authorities](https://docs.openshift.com/container-platform/4.12//rest_api/config_apis/proxy-config-openshift-io-v1.html#spec-trustedca) required to validate the proxy's certificate
 * Configure the proxy settings in WMCO-managed components on Windows nodes (kubelet, containerd runtime)
 * React to changes to the cluster-wide proxy settings during WMCO runtime
+* Synchronize environment variables `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` on Windows nodes with cluster-wide 
+  proxy settings in both shells _CMD_ and _PowerShell_
```

Comment:
I feel that this point conflicts with the PowerShell non-goals: PowerShell's shell sessions do not inherit proxy settings by default on Windows nodes. Additionally, do we specifically do anything for it to be synchronized in both shells?

Reply:
Re-worded it for more clarity, but it does not. This refers to the environment variables. 

The  PowerShell non-goals, cover enabling proxy by default in Powershell session, where the environment variables should exist, as per goals.